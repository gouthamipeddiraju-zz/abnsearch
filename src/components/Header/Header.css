.headerWrapper {
  background-color: #1976d2;
  height: 70px;
  text-align: center;
  border: 1px solid #999;
  box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2),
    0px 4px 5px 0px rgba(0, 0, 0, 0.14), 0px 1px 10px 0px rgba(0, 0, 0, 0.12);
}

.headerText {
  color: #fff;
  font-size: 23px;
  padding: 20px;
}
