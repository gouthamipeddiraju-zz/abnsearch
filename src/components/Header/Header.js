import React from "react";
import './Header.css';

class Header extends React.Component {
    render() {
      return (
        <div className="headerWrapper">
          <div className="headerText">ABN Search tool</div>
        </div>
      );
    }
}

export default Header;