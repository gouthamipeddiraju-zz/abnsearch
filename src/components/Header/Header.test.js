import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header component', () => {
    it('Test rendering', () => {
        const tree = renderer.create(<Header/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});