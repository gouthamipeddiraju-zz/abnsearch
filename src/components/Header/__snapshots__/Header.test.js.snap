// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Test rendering 1`] = `
<div
  className="headerWrapper"
>
  <div
    className="headerText"
  >
    ABN Search tool
  </div>
</div>
`;
